_data/speakers.yml

```diff
@@ -328,15 +340,16 @@
   image: Mikkel_Lykkegaard.JPG
   thumbnail: Mikkel_Lykkegaard.JPG
   alt: Mikkel
-  topic:
-  proposal_summary:
+  topic: The MLDA multilevel sampler in PyMC3
+  proposal_summary: This presentation will give you the chance to know more about PyMC3’s new multilevel MCMC sampler, MLDA, and help you use it in practice. MLDA exploits multilevel model hierarchies to improve sampling efficiency compared to standard methods, especially when working with high-dimensional problems where gradients are not available. We will present a step-by-step guide on how to use MLDA within PyMC3, go through its various features and also present some advanced use cases, e.g. employing multilevel PDE-based models written in FEniCS and using adaptive error correction to correct model bias between different levels.
   discourse_link:
   youtube_link:
   description: Mikkel Lykkegaard is a PhD student with the Data Centric Engineering Group and Centre for Water Systems (CWS) at University of Exeter.
     His research is mainly concerned with Uncertainty Quantification (UQ) for computationally intensive forward models.
   presentation_type: Talk
-  time_zone:
+  time_zone: Asia/Africa/Europe
   track: Advanced
+  keywork: Mikard
```

Comment:
mlda multilevel sampler has 3 speakers, they should all have the same keyword.

Reply:
Done this